src/core/hyprlock.cpp

```diff
@@ -1156,7 +1226,7 @@ zwlr_screencopy_manager_v1* CHyprlock::getScreencopy() {
 }
 
 void CHyprlock::attemptRestoreOnDeath() {
-    if (m_bTerminate || m_sCurrentDesktop != "Hyprland")
```

Comment:
Any Idea why we didn't try to restore when `m_bTerminate` is true before?

Reply:
Maybe we want to check if we are actually locked instead?
Cause if we aren't there is no point in trying to restore.

Reply:
uhhh cuz if it crashes after nlock

Reply:
yeah but terminate does not mean we already unlocked

Reply:
oh then ok